Covinator/Covinator/Pages/PfizerVaccination.cshtml

```diff
@@ -23,12 +25,23 @@
     }
 </style>
 <div class="text-center">
-    <h1 class="display-4">Pfizer Vaccine Allocations For All States: <h3>@currentWeek</h3></h1>
+    <h1 class="display-4">Pfizer Vaccine Allocations: </h1>
+    
+    <br />
+    @if (@jurisdiction is null)
+    {
+        <h3> Vaccine distribution on @currentWeek.ToString().Substring(0, 9) by states</h3>
+    }
+    else
+    {
+        <h3> For the State of : @jurisdiction</h3>
+    }
+
```

Comment:
The heading “Vaccine distribution on (current week) by states” was same even after selecting a particular state. I made the changes to show the heading state wise.